scripts/fedora_install.sh

```diff
@@ -19,6 +19,7 @@ dnf upgrade -y && dnf install -y \
   net-tools \
   tcpreplay \
   python3-virtualenv \
+  gawk \
```

Comment:
Nitpick but better to keep this sorted

Reply:
Fixed in this commit: https://github.com/seladb/PcapPlusPlus-DockerImages/pull/53/commits/121fb5a4fe6a6a78afdc949a7b8c010faa387efd